packages/flutter_localizations/lib/src/l10n/cupertino_ar.arb

```diff
@@ -42,6 +42,6 @@
   "tabSemanticsLabel": "علامة التبويب $tabIndex من $tabCount",
   "modalBarrierDismissLabel": "رفض",
   "searchTextFieldPlaceholderLabel": "بحث",
-  "noSpellCheckReplacementsLabel": "No Replacements Found",
+  "noSpellCheckReplacementsLabel": "لم يتم العثور على بدائل.",
```

Comment:
```suggestion
  "noSpellCheckReplacementsLabel": "لم يتم العثور على بدائل",
```
No need for this extra dot(.) in the Arabic translations

This is from Native IOS:
![IMG_4702](https://github.com/flutter/flutter/assets/40350360/cb2dfce5-4605-4061-983f-698d4bb06054)

Reply:
I requested a change to the translations internally.